sooyoung/app.js

```diff
@@ -0,0 +1,42 @@
+const express = require("express");
+const dotenv = require("dotenv");
+const cors = require("cors");
+const morgan = require("morgan");
+const { DataSource } = require('typeorm');
+
+dotenv.config();
+
```

Comment:
모듈을 import 해오는 부분을 잘 작성해주셨네요!

모듈은 third-party 모듈 / built-in 내장 모듈 / custom 모듈이 존재합니다.

모듈을 Import 하는 코드의 순서를 조정하여 가독성을 높일 수 있는데요. 다음과 같이 작성할 수 있습니다.
1. built-in module
2. third-party module
3. custom module

현재 코드 상에선 built-in 모듈과 custom 모듈이 존재하지 않으니 아래와 같이 수정할 수 있습니다.
```suggestion
// third-party module
require('dotenv').config(); // 환경변수 사용

const express = require('express');
const cors = require('cors');
const morgan = require('morgan');
const { DataSource } = require('typeorm');
```